tree/dataframe/src/RDataFrame.cxx

```diff
@@ -381,6 +382,78 @@ ROOT::EnableImplicitMT();
 ~~~
 Simple as that. More details are given [below](#parallel-execution).
 
+
+##  <a name="python"></a>Efficient analysis in Python
+
+You can use `RDataFrame` in Python due to the dynamic C++/Python translation of PyROOT. In general, the interface
+is the same than for C++, a simple example follows.
+
+~~~{.python}
+df = ROOT.RDataFrame("myTree", "myFile.root")
+sum = df.Filter("x > 10").Sum("y")
+print(sum.GetValue())
+~~~
+
+### Simple usage of efficient C++ code in Python
+
+To perform more complex operations in the `RDataFrame` graph, e.g., in `Filter` and `Define` nodes, which don't
+fit into a simple expression string, you can just-in-time compile such functions directly in the Python script
+via the C++ interpreter cling. This approach has the advantage that you get the efficiency of compiled C++ code
+combined with the convenient workflow of a Python script. See the following snipplet for an example of how to
```

Comment:
typo: snipplet -> snippet

Reply:
I just learnt that snipplet is not an English word :D Thanks!